packages/grafana-ui/src/components/Select/HeadlessSelect.tsx

```diff
@@ -0,0 +1,8 @@
+import React, { FunctionComponent } from 'react';
+import Select, { CommonProps, SelectProps } from './Select';
+
+export const HeadlessSelect: FunctionComponent<CommonProps & SelectProps> = props => (
```

Comment:
React-Select doesn't give you an option to hide input part of the select so that's why I created the HeadlessSelect.
Yes, I was going to do that but I felt that time was slipping away. Makes more sense to group them.

Reply:
OK, I understand. But what happens in react-select if you pass `null` for a given component? Is it even allowed (ts)?

Reply:
I couldn't find the right component to pass null or it crashed.

Reply:
Could you check TimePicker and RefreshPicker again @dprokop ? Thanks